TelCo.ColorCoder/Colours.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class Colours
+    {
+        public static Color[] colorMapMajor, colorMapMinor;
+        internal Color majorColor, minorColor;
+        public override string ToString()
+        {
+            return string.Format("MajorColor:{0}, MinorColor:{1}", majorColor.Name, minorColor.Name);
+        }
+        static Colours()
+        {
+            colorMapMajor = new Color[] { Color.White, Color.Red, Color.Black, Color.Yellow, Color.Violet };
+            colorMapMinor = new Color[] { Color.Blue, Color.Orange, Color.Green, Color.Brown, Color.SlateGray };
+        }
+
+        public static int getMajorIndex(ColorPair pair)
```

Comment:
How does base class can aware of existence of Derived Class ?

Reply:
This was a blunder, rectified it now,